Add ModuleInfo for torch.nn.ChannelShuffle

isdanni opened the pull request:
Summary:
Add ModuleInfo for torch.nn.ChannelShuffle:

https://github.com/pytorch/pytorch/pull/105351#pullrequestreview-1533879586

Test Plan: Please see GitHub Actions.

Differential Revision: D48021100

isdanni commented:
@pytorchbot label "topic: not user facing"

mikaylagawarecki commented on torch/testing/_internal/common_modules.py:
```diff
@@ -771,6 +771,22 @@ def module_inputs_torch_nn_BatchNorm3d(module_info, device, dtype, requires_grad
+    make_input = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+
+    return [
+        ModuleInput(constructor_input=FunctionInput(2),
```
This sample input where you pass in a singleton tensor is causing the failure, seems like there is a `TORCH_CHECK` that ensures input size is at least 2 [here](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/ChanelShuffle.cpp#L36-L38)

mikaylagawarecki reviewed (commented):
Thanks @isdanni, added a comment on how to fix CI :)

isdanni committed:
Add ModuleInfo for torch.nn.ChannelShuffle (#106525)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/106525

Add ModuleInfo for torch.nn.ChannelShuffle:

https://github.com/pytorch/pytorch/pull/105351#pullrequestreview-1533879586

Test Plan: Please see GitHub Actions.

Differential Revision: D48021100

fbshipit-source-id: d00d20c631e9d8f261239305343d07ae70f55b54

isdanni commented:
So sorry for the delay! I'm working on fixing this. The Internal Sync check might be a bit hard to bypass. If it's not possible to fix I will open a new one.
